stdpopsim/cli.py

```diff
@@ -544,7 +545,7 @@ def run_simulation(args):
                     "and the model described in the original publication. "
                     "More information about the QC process can be found in "
                     "the developer documentation. "
-                    "https://stdpopsim.readthedocs.io/en/latest/development.html"
+                    "https://popsim-consortium.github.io/stdpopsim-docs/stable/development.html"  # noqa: E501
```

Comment:
Maybe developer docs should point to /latest/?

Reply:
yes I would like this!